problem_builder/mentoring.py

```diff
@@ -214,6 +225,10 @@ def author_preview_view(self, context):
         self.include_theme_files(fragment)
         return fragment
 
+    def max_score(self):
+        """ Maximum score. We scale all scores to a maximum of 1.0 so this is always 1.0 """
+        return 1.0
```

Comment:
How does this relate to the weight field above  Is there a difference between score and grade?